### PR TITLE
Fixed TypesToLoadAttribute compatibility

### PR DESCRIPTION
**Pull request by Haplois**
TestFx updated to implement changes in https://github.com/microsoft/vstest/pull/2674; this should fix the compatibility issue introduced in https://github.com/microsoft/vstest/pull/2537 and https://github.com/microsoft/testfx/pull/739.

### CODE REVIEW

**Review by peterwald — APPROVED** [review]

File: src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities
+{
+    /// <summary>
+    /// Custom Attribute to specify the exact types which should be loaded from assembly
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = false)]
+    internal sealed class TypesToLoadAttribute : Attribute
```
Comment (shyamnamboodiripad) [inline src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs:17]: I'd [mentioned](https://github.com/microsoft/testfx/pull/746#discussion_r533752933) this to @Sanan07, but perhaps `TestExtensionTypesAttribute` would be be a better name (i.e. a bit less confusing for adapter implementers).

File: src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
+
+[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities
```
Comment (shyamnamboodiripad) [inline src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs:11]: Could we make the namespace simply `Microsoft.VisualStudio.TestPlatform` or `Microsoft.VisualStudio.TestPlatform.ObjectModel`? The `Utilities` part of the name seems verbose and I may not intuitively remember that the attribute needs to be created in that namespace...

**Review by shyamnamboodiripad — COMMENTED** [review]
LGTM modulo comments about naming

File: src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs (hunk 1 above)
Comment (Haplois) [reply src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs:17]: That's a good idea and a relatively simple change. I'll do it tomorrow morning.

**Pull request merged**